datalad/distribution/dataset.py

```diff
@@ -223,6 +223,27 @@ def get_subdatasets(self, pattern=None, fulfilled=None, absolute=False,
 #        """
 #        raise NotImplementedError("TODO")
 
+    # TODO maybe needs to get its own interface
+    def remember_state(self, message, auto_add_changes=True, version=None):
```

Comment:
so it is the "save" functionality?   it is not tested BTW -- so I guess not used... worth having?